test/resources.go

```diff
@@ -116,9 +116,25 @@ func NewFlightRecorderBadJMXPassKey() *rhjmcv1beta1.FlightRecorder {
 
 func newFlightRecorder(jmxAuth *rhjmcv1beta1.JMXAuthSecret) *rhjmcv1beta1.FlightRecorder {
 	return &rhjmcv1beta1.FlightRecorder{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "FlightRecorder",
+			APIVersion: "rhjmc.redhat.com/v1beta1",
+		},
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      "test-pod",
-			Namespace: "default",
+			Name:            "test-pod",
+			Namespace:       "default",
+			ResourceVersion: "2",
```

Comment:
You should probably leave out ResourceVersion as this will vary by test, and compare Name, Namespace, Labels, OwnerReferences separately.

Reply:
Done!